.github/ISSUE_TEMPLATE/site_showcase.md

```diff
@@ -0,0 +1,102 @@
+---
+name: Site Showcase Submission
+about: Want to submit a site to the site showcase? Follow these instructions.
+---
+
+<!--
+  To add your site to the showcase, please follow the suggested format below.
+
+  Before submitting a site to the site showcase, please make sure no on else has already submitted it by searching existing PRs (if there is a chance that someone else could have submitted it): https://github.com/gatsbyjs/gatsby/pulls
+-->
+
+# Steps
+
+There is only one major step :)
+
+1. Edit gatsby/docs/sites.yml by adding your submission to the bottom of the list of sites in the following format:
```

Comment:
Wondering if this should this be `gatsby/docs/sites.yml` or just `docs/sites`?

Reply:
Also we could link "Edit docs/sites.yml". I think https://github.com/gatsbyjs/gatsby/blob/master/docs/sites.yml should be correct?

/cc @m-allanson @pieh

Reply:
Ah, seems we can directly link to the "edit" screen with https://github.com/gatsbyjs/gatsby/edit/master/docs/sites.yml

Reply:
Here's a link pointing to edit `sites.yml` at @ThatOtherPerson's working branch for the site showcase: https://github.com/ThatOtherPerson/gatsby/edit/topics/showcase-rebase-v2/docs/sites.yml

Reply:
Hmmm, currently in v2 there is `sites.yml` that people seem to put sites into in https://github.com/gatsbyjs/gatsby/blob/v2/www/src/data/sites.yml (in `www/src/data` dir not in `docs` like You link @fk - which one should be final?

Reply:
I'm still unclear on which file people need to edit. Once I have that, I think I'll just make a doc, like @m-allanson suggested! It will only take like 5 minutes!